assets/common/components/deploy_model/spec.yaml

```diff
@@ -8,9 +8,7 @@ is_deterministic: True
 display_name: Deploy model
 description:
   Deploy a model to a workspace. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
-
 environment: azureml://registries/azureml/environments/python-sdk-v2/versions/15
-
```

Comment:
why are we removing empty lines? Please revert it, if not needed.

Reply:
reverted